src/custom/components/AccountDetails/Transaction/index.tsx

```diff
@@ -182,23 +186,23 @@ export default function Transaction({ hash: id }: { hash: string }) {
     ? new Date(Date.parse(creationTimeOrder))
     : undefined
 
-  const timeFormatOptionMDY: Intl.DateTimeFormatOptions = {
-    dateStyle: 'long',
-  }
+  // const timeFormatOptionMDY: Intl.DateTimeFormatOptions = {
+  //   dateStyle: 'long',
+  // }
 
   const timeFormatOptionHM: Intl.DateTimeFormatOptions = {
     timeStyle: 'short',
   }
 
   // Month Day, Year
-  const creationTimeMDY = creationTimeFull?.toLocaleString(undefined, timeFormatOptionMDY)
+  // const creationTimeMDY = creationTimeFull?.toLocaleString(undefined, timeFormatOptionMDY)
 
   // Hour:Minute
   const creationTime = creationTimeFull?.toLocaleString(undefined, timeFormatOptionHM)
 
   return (
     <Wrapper>
-      {creationTimeMDY && <CreationDateText>{creationTimeMDY}</CreationDateText>}
+      {/*{creationTimeMDY && <CreationDateText>{creationTimeMDY}</CreationDateText>}*/}
```

Comment:
This component is for the individual activity. Setting the date here would make all entries to have it, so I only set it on the parent